nixos/modules/image/repart-image.nix

```diff
@@ -0,0 +1,75 @@
+{ name
```

Comment:
Please add a comment that describes that this is a callPackage definition called only from repart.nix and NOT a module that you can import.

Reply:
added